dashboards-observability/public/components/visualizations/charts/maps/treemap_type.ts

```diff
@@ -67,6 +68,12 @@ export const createTreeMapDefinition = (params: BarTypeParams = {}) => ({
         editor: ConfigEditor,
         content: [],
       },
+      {
+        id: 'availability-panel',
+        name: 'Availability',
+        mapTo: 'availabilityConfig',
+        editor: ConfigAvailability,
+      },
```

Comment:
How do users apply availability to tree map?

Reply:
The tab is visible but the button to add availability will only be enabled for line and bar type

Reply:
Looks like we have it for all visualizations, is this desired?

Reply:
Talked to Kevin and you are right, we should just hide it for non-applicable chart types.